Resources/Prototypes/_ES/Catalog/Crates/crates.yml

```
@@ -0,0 +1,279 @@
+#Engineering
+- type: entity
+  parent: CrateElectrical
+  id: ESCrateEngineeringCableBulk
+  name: bulk cable crate
+  description: 2 coils each for every cable type.
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:AllSelector
+        children:
+        - id: CableHVStack
+          amount: 2
+        - id: CableMVStack
+          amount: 2
+        - id: CableApcStack
+          amount: 2
+          
+- type: entity
+  parent: CrateGenericSteel
+  id: ESCrateMaterialGlass
+  name: glass sheet crate
+  description: 90 sheets of glass, packed with care.
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage:
+        id: SheetGlass
+        amount: 3
+        
+- type: entity
+  parent: CrateGenericSteel
+  id: ESCrateMaterialSteel
+  name: steel sheet crate
+  description: 90 sheets of steel.
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage:
+        id: SheetSteel
+        amount: 3
+        
+- type: entity
+  parent: CrateGenericSteel
+  id: ESCrateMaterialPlastic
+  name: plastic sheet crate
+  description: 90 sheets of plastic.
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage:
+        id: SheetPlastic
+        amount: 3
+        
+- type: entity
+  parent: CrateGenericSteel
+  id: ESCrateMaterialPlasteel
+  name: plasteel crate
+  description: 30 sheets of plasteel.
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage:
+        id: SheetPlasteel
+
+- type: entity
+  parent: CrateEngineeringSecure
+  id: ESAdvancedToolsCrate
+  name: advanced tool crate
+  description: With these you might almost feel ready for a promotion!
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:AllSelector
+        children:
```

```suggestion
      entity_storage:
        all:
```
You can do this to shorten it a bit and make it easier to read (this can be done across most of this file)